doc/nrf/ug_nrf9160.rst

```diff
@@ -24,33 +24,94 @@ For more details on the SiP, see the `nRF9160 product website`_.
 
 The figure illustrates the conceptual layout when targeting
 an nRF9160 Cortex-M33 application MCU with TrustZone.
-The M33 TrustZone divides the MCU into secure and non-secure domains.
-When the MCU boots, it always starts executing from the secure area,
-which in this case is the **Secure Partition Manager**.
-The **Secure Partition Manager** configures a part of memory and peripherals to be
-non-secure and then jumps to the main application located in the non-secure area.
-The **BSD Socket** library must be embedded in the application because
-BSD sockets are the primary interface to the LTE modem.
+
+Application MCU
+===============
+
+The M33 TrustZone divides the application MCU into secure and non-secure domains.
+When the MCU boots, it always starts executing from the secure area.
+The secure bootloader chain starts the Secure Partition Manager, which configures a part of memory and peripherals to be non-secure and then jumps to the main application located in the non-secure area.
+
+The :ref:`nrf9160_pca10090` board in Zephyr provides two different targets:
```

Comment:
Is this terminology agreed upon? As seen from zephyr these two "targets" are actually two completely different boards.

Reply:
Rephrased.